src/modules/ui/components/action_logger/index.js

```diff
@@ -34,12 +34,60 @@ const btnStyle = {
   marginLeft: 5,
 };
 
+const countStyle = {
+  display: 'inline-block',
+  backgroundColor: '#777777',
+  color: '#ffffff',
+  padding: '1px 5px',
+  borderRadius: '8px',
+};
+
+const logDivStyle = {
+  margin: 5,
+  paddingBottom: 3,
+  marginBottom: 4,
+  overflow: 'auto',
+  borderBottom: '1px solid #fafafa',
+};
+
+const inspectorStyle = {
+  marginLeft: 5,
+  float: 'none',
+  display: 'inline-block',
+};
+
+const countWrapper = {
+  minWidth: 20,
+  display: 'inline-block',
+  float: 'left',
+  height: 19,
+  marginRight: 5,
+};
+
+const actionNameStyle = {
+  float: 'right',
+  padding: '0 5px',
+  fontWeight: 'bold',
+};
+
 class ActionLogger extends Component {
 
   getActionData() {
     return this.props.actions
       .map((action) => (
-        <Foldable key={action.id} action={action} />
+      <div style={logDivStyle} key={action.id}>
+        <div style={countWrapper}>
+          { action.count > 1 && <span style={ countStyle }>{ action.count }</span> }
+        </div>
+        <div style={inspectorStyle}>
+          <Inspector
+            showNonenumerable
+            name={action.data.name}
+            data={action.data.args || action.data}
+          />
+        </div>
+        <span style={actionNameStyle}>{action.data.name}</span>
```

Comment:
@ritz078 When we update the new inspector. We may need to remove this right?
Could you work on that?